config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -8,10 +8,21 @@ en:
             my_student_header: 'My Student Submissions'
             student_header: 'Student Submissions'
             other_header: 'Other Submissions'
+            publish: 'Publish Grades'
+            confirm_publish: |
+
+              THIS ACTION IS IRREVERSIBLE
+
+              All students will see their grades.
```

Comment:
`All graded submissions will be published and students will be able to see their own published grades.` 

How about this? `students will see the grades` can be rather scary, in the sense that can student A see student B's published grades?